core/spec/models/spree/payment_spec.rb

```diff
@@ -1207,51 +1207,49 @@
     end
   end
 
-  describe "is_avs_risky?" do
+  describe "risky?" do
```

Comment:
I think there're some conditions that we're not testing, like the `failed` state. However, as I said in the other comment, I'm not sure I understand why we need the method.

Reply:
See above, now it's being used elsewhere, and properly tested 👍 good call